sdk/communication/Azure.Communication.CallingServer/tests.yml

```diff
@@ -15,9 +15,9 @@ extends:
         SubscriptionConfigurations:
           - $(sub-config-communication-int-test-resources-common)
           - $(sub-config-communication-int-test-resources-net)
-    Clouds: Public,Int
+    Clouds: Public
     EnvVars:
       # SKIP_PHONENUMBER_LIVE_TESTS skips certain phone number tests such as purchase and release
       SKIP_PHONENUMBER_LIVE_TESTS: TRUE
       # SKIP_CALLINGSERVER_INTERACTION_LIVE_TESTS skips certain callingserver tests that required human interaction
-      SKIP_CALLINGSERVER_INTERACTION_LIVE_TESTS: TRUE
\ No newline at end of file
+      SKIP_CALLINGSERVER_INTERACTION_LIVE_TESTS: TRUE
```

Comment:
Do you still need this flag? SKIP_CALLINGSERVER_INTERACTION_LIVE_TESTS  now that you remove the INT from Clouds config?

Reply:
Yes we still need it. This flag is used to skip tests required human Interaction(like pick up the call) Which cannot be perform in Live Mode test pipeline.